registrations/tests.py

```diff
@@ -2927,7 +2928,7 @@ def test_registration_process_pmtct_no_facility(self):
         Registration.objects.create(**registration_data)
 
         # check jembi registration
-        jembi_call = responses.calls[13]  # jembi should be the fourteenth one
+        jembi_call = responses.calls[9]  # jembi should be the tenth one
```

Comment:
All these where the calls are less doesn't look right.

Reply:
may my above comment relate to this as well?

Reply:
Can we double check this one please?

I recon it's because it wasn't mocked so it retried 5 times.